manager/testdata/new-blueprints/write-module.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: app.m4d.ibm.com/v1alpha2
+kind: M4DModule
+metadata:
+  name: arrow-flight-common-write-conf
+  namespace: m4d-system
+  labels:
+    version: 0.0.1
+spec:
+  description: "This module knows how to configure a cluster wide arrow flight service"
+  # this field is true if this module serves multiple users. Otherwise it
+  # is false. It is used for example when setting the endpoint to
+  # access the module service - if it is a multi-user module then
+  # the service name will be fixed and will be taken from
+  # M4DModule.Capabilities.Api.endpoint.host.
+  multi-user: true
+  confChart:
+    name: ghcr.io/mesh-for-data/afwm:0.1.0
+    values: {}
```

Comment:
This module will deploy the arrow-flight server in a lazy fasion, right? i.e., it a configuration module but also deploys the chart if it is not running. in case the server was deployed already (not by m4d for example) I guess that the chart name will be omitted.

Reply:
No there is no lazy fashion. (At least for now I would suggest) The service will be installed by the admin when the `M4DModule` CRD will be installed in each cluster. The chart is containing a K8s object that configures the service. (Be it a config map or a pod that sends a curl command to the service)